default.yml

```diff
@@ -11,7 +11,6 @@
     os_desktop_enable: true
     os_env_extra_user_paths: ['/home']
     os_auth_allow_homeless: true
-    os_security_kernel_enable_core_dump: true
```

Comment:
I removed it as it's a duplicate - the other being on line 8.  Happy to set to false if you want - but not sure how this file interacts with defaults/main.yml, i.e. which takes priority?